test/Microsoft.AspNet.Cors.Test/CorsTestFixtureOfT.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.TestHost;
+
+namespace Microsoft.AspNet.Cors.Infrastructure
+{
+    public class CorsTestFixture<TStartup> : IDisposable
+        where TStartup : class
+    {
+        private readonly TestServer _server;
+
+        public CorsTestFixture()
+        {
+            var builder = new WebHostBuilder().UseStartup<TStartup>();
+            _server = new TestServer(builder);
```

Comment:
Couldn't find a `TestServer.Create<TServer>` method so I did this two-step instead.